packages/builder/src/components/AlloWrapper.tsx

```diff
@@ -21,35 +21,46 @@ function AlloWrapper({ children }: { children: JSX.Element | JSX.Element[] }) {
   const [backend, setBackend] = useState<Allo | null>(null);
 
   useEffect(() => {
-    console.log("AlloWrapper: useEffect", web3Provider, signer, chainID);
     if (!web3Provider || !signer || !chainID) {
       setBackend(null);
     } else {
-      // const alloBackend: Allo = new AlloV2({
-      //   chainId: chainID,
-      //   transactionSender: createEthersTransactionSender(signer, web3Provider),
-      //   projectRegistryAddress: "0x0000000000000000000000000000000000000000",
-      //   ipfsUploader: createPinataIpfsUploader({
-      //     token: getConfig().pinata.jwt,
-      //     endpoint: `${getConfig().pinata.baseUrl}/pinning/pinFileToIPFS`,
-      //   }),
-      //   waitUntilIndexerSynced: waitForSubgraphSyncTo,
-      // });
-
       const addresses = addressesByChainID(chainID!);
+      const config = getConfig();
+      let alloBackend: Allo;
+
+      if (config.allo.version === "allo-v2") {
+        alloBackend = new AlloV2({
+          chainId: chainID,
+          transactionSender: createEthersTransactionSender(
+            signer,
+            web3Provider
+          ),
+          projectRegistryAddress: "0x0000000000000000000000000000000000000000",
```

Comment:
this doesn't seem right 👀

Reply:
I noticed the projectRegistryAddress is not used in allo v2, we can remove it from the constructor